web/Web/web/review/templates/review/review.html

```diff
@@ -0,0 +1,293 @@
+                                data-target="#additionalJudgingCriteriaModal" href="#"><i
+                                    class="fas fa-info-circle"></i></a></h6>
+                        {% include 'cards/configurable_criteria_card.html' %}
+                        {% endif %}
+
+                    </div>
+
+                    <div id="highlighted_words">
+                        <h6 class="card-title text-secondary mt-4 unselectable" unselectable="on">Highlighted terms
+                            <span class="badge badge-secondary float-right">Beta</span></h6>
+                        <ul class="list-group" id="highlighted_terms_list"></ul>
+                    </div>
+
+                </div>
+            </div>
+        </div>
+
+        <div class="col-2 d-none d-sm-block">
+            <div class="sticky-top card border-0 mr-2" style="top:75px; z-index: 0;">
+                <div class="card-body p-0 mt-3">
+                    <nav class="nav nav-pills small flex-row flex-sm-row unselectable" unselectable="on">
+                        <a class="flex-sm-fill text-sm-center nav-link active">Reviewed Documents</a>
+                        <button class="btn btn-sm sort_by_relevance" type="button">&darr;</button>
+                    </nav>
+
+                    <div class="tab-content">
+                        <div class="tab-pane fade show active" id="reviewed-documents-tab" role="tabpanel">
+                            <div class="mt-3" id="latest_judged">
+                                <div class="card border-0">
+                                    <div class="card-body px-0 py-2 d-flex"
+                                        style="min-height: 18rem; max-height: 38rem;">
+                                        <!-- loader -->
+                                        <div class="mx-auto previouslyReviewedListSpinner mt-5">
+                                            <div class="la-ball-circus text-secondary">
+                                                <div></div>
+                                                <div></div>
+                                                <div></div>
+                                                <div></div>
+                                                <div></div>
+                                            </div>
+                                        </div>
+                                        <!-- end loader -->
+                                        <div id="previouslyReviewedList"
+                                            class="overflow-auto small previouslyReviewedList" style="min-width: 0">
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+
+                        {% if "scal" in request.user.current_session.strategy %}
```

Comment:
We don't need the this information in the review interface

Reply:
Removed